### ISSUE TITLE
When declarations with multiple supporting document are sent offline, the record disappears later when go online

### ISSUE THREAD

**Issue by SyedaAfrida** [opencrvs/opencrvs-core#5313]
**Bug description:**
When declarations with multiple supporting document are sent offline, the record disappears later when go online

**Steps to reproduce:**
1. login to FA
2. Go offline
3. Declare any event
4. Attach multiple supporting documents 
5. Submit offline
6. Go online and observe the record in outbox and other tabs

**Actual result:**
The record disappear after being sent from outbox

**Expected result:**
The record should not disappear

**Screen recording:**
For marriage in real device
 
https://drive.google.com/drive/folders/1P4oeP1ifD7SHbbocSRA5dh499mZKOXe2

For death in real device

https://drive.google.com/drive/folders/1P4oeP1ifD7SHbbocSRA5dh499mZKOXe2

**Tested on:**
https://login.farajaland-qa.opencrvs.org/?lang=en

**Comment by euanmillar** [comment]
@anny320 @SyedaAfrida I dont think that a drive folder is right location for the videos on this ticket.  They should be accessible to us in Loom.  These appear to be personal drive links that we dont have access to

**Comment by anny320** [comment]
please check now @euanmillar

**Comment by Zangetsu101** [comment]
We weren't able to reproduce this
